data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditableAuthorisedDataRequest.java

```diff
@@ -22,16 +22,16 @@
 import java.util.StringJoiner;
 
 /**
- * The class contains the authorised access to the resource.  This class is a container for the original data,
- * {@link DataRequest}, the reference to the data that is authorised to be seen, {@link AuthorisedDataRequest} and
- * {@link AuditErrorMessage} processing.  The container will hold the {@code AuditErrorMessage} if there was an error
- * in getting the authorised data.
+ * The class contains the data available after the request to authorise the access to the data. This will include the
```

Comment:
I think this needs a minor adjustment so that it reads better. The "contains the data available" is a bit misleading.
Maybe something like this:
```suggestion
 * The class contains the information about whether the read request is authorised. This will include the
```